common/subdevice-model.cpp

```diff
@@ -81,6 +81,18 @@ namespace rs2
             auto opt = static_cast<rs2_option>(i);
 
             opt_container[opt] = create_option_model(opt, opt_base_label, model, options, options_invalidated, error_message);
+            model->s->on_options_value_update( [model]( const options_list & list )
+            {
+                for( auto opt_id : list )
+                {
+                    auto it = model->options_metadata.find( opt_id );
+                    if( it != model->options_metadata.end() &&
+                        model->options_metadata[opt_id].endpoint->supports( opt_id ) )
```

Comment:
Why search again instead of using the iterator?

Reply:
Is it possible that an option was changed that isn't supported?

Reply:
The `options_list` comes from the device, second search is in the model database.
At start up options from the device can update before they are registered at the model so I need to check.

Removed the `supports`, leftover from previous implementation.

Reply:
I'm saying `model->options_metadata` is searched twice.

Reply:
Fixed, using iterator